src/main/java/frc/robot/OI.java

```diff
@@ -59,6 +67,14 @@ public OI() {
     ////////////////////////////////////////////////////
 
 
+    ////////////////////////////////////////////////////
+    // Now Mapping Commands to Keyboard Box
+    ////////////////////////////////////////////////////
+
+    keyboardBox.getKForward().whileHeld(new KeyboardDrive(0.75, 0));
```

Comment:
Perhaps we can give the constants names (defined in KeyboardDrive) and pass in the named constants